hooks/useRebalanceHandler.tsx

```diff
@@ -0,0 +1 @@
+export const useRebalanceHandler = () => {}
```

Comment:
Did you mean to do this? Or are you adding the logic for next PR? If so, remove this file for now

Reply:
Since this PR is only for th UI, this file is just a placeholder for the future backend logic.